sdk/core/System.ClientModel/samples/ModelReaderWriter.md

```diff
@@ -53,4 +53,47 @@ string json = @"{
       ""z"": 3
     }";
 OutputModel? model = JsonSerializer.Deserialize<OutputModel>(json, options);
+```
+
+## Using a Proxy
+
+In more advanced scenarios a library user might want to override the behavior of how a model is read or written.
+In this case you can implement your own class which implements the same interface, either `IPersistableModel<T>` or `IJsonModel<T>`, and register it with the `ModelReaderWriterOptions`.
```

Comment:
Would it be possible to call out the difference between the new proxy feature and `[PersistableModelProxy]` here?  I expect the reuse of that name to be confusing to customers so it would be nice to provide reference materials to help them understand.